File: express/app/src/main/java/com/changsukuaidi/www/base/i/IBaseView.java
package com.changsukuaidi.www.base.i;

/**
 * @Describe
 * @Author zhiyicx
 * @Date 2017/12/1
 * @Contact
 */
public interface IBaseView {

    void showSuccessMessage(String message);

    void showFailedMessage(String message);

    void showLoading(String msg);

    void showLoading();

    void hideLoading();

    void onNetError();

    void showEmpty();
}

File: WIFIDirect/app/src/main/java/com/thc/www/p2p/base/IBaseView.kt
package com.thc.www.p2p.base

/**
 * @Describe
 * @Author thc
 * @Date  2018/4/20
 */
interface IBaseView {
    fun showSuccessMessage(message: String)

    fun showFailedMessage(message: String)

    fun showLoading(msg: String)

    fun showLoading()

    fun hideLoading()

    fun onNetError()

    fun showEmpty()

    fun finishActivity()
}